### DISCUSSION TITLE
Snap to grid in the skin editor

### DISCUSSION THREAD

**Discussion by Natelytle** [ppy/osu#16533] [Ideas] [answered]
At the moment it's frustrating to use the skin editor due to the fact that there isn't any snapping or guard rails in place to help position skin elements. Trying to put the HP bar exactly in the corner is a challenge, same with aligning the pp counter with the score counter. My proposal is to add a snap to grid toggle to the skin editor that snaps the elements to the grid, and potentially gives guardrails to help snap elements to each other.

**Comment by Natelytle** [comment, marked as answer]
My bad, there is already an answer at #15044

**Reply by mk56-spn** [reply]
partially my fault i forgot to tell him to check for previous issues
